tests/test_read_edf.py

```diff
@@ -0,0 +1,15 @@
+from pathlib import Path
+
+from electos.ballotmaker import read_edf
+
+test_dir = Path(__file__).parent.resolve()
+# this test file must be in the same dir as this test script
+test_file = Path("june_test_case.json")
+full_test_path = Path(test_dir, test_file)
+
+
+def test_read_edf():
+    # is the test file available?
+    assert Path.is_file(full_test_path)
```

Comment:
As above: `Path.is_file` is an instance method not a class method:

```suggestion
    assert full_test_path.is_file()
```

Reply:
Fixed